### PR TITLE
Update patch for roberta-large-mnli in third_party_integration

### PR DESCRIPTION
**Pull request by AlexanderDokuchaev**
### Changes

- Remove `attention_mask` from `input_info` to disable tracing. The placement of the FQ has not changed.
- Bump version of transformers and optimum

### Reason for changes 

After changes in the graph trace, `attention_mask` branch was added in NNCFgraph as result added FQ for mask. 
But since the mask is used values 0 and finfo(float32).min, inf and nan appear at backpropagation, as result parameters xof FQ are set nan.

### Related tickets

127936

### Tests

test_sanity_third_party.TestTransformers.test_glue_train

### CODE REVIEW

**Comment by AlexanderDokuchaev** [comment]
ubuntu20_3dparty/285/

File: tests/post_training/requirements.txt [hunk 1]
```diff
@@ -2,14 +2,14 @@
 torch==2.1.0
 --extra-index-url https://download.pytorch.org/whl/cpu
 torchvision==0.16.0
-transformers==4.30.0
+transformers==4.36.0
```
Comment (MaximProshin) [inline tests/post_training/requirements.txt:5]: please use ~=4.36 instead

File: tests/torch/pruning/experimental/test_nodes_grouping.py [hunk 2]
```diff
@@ -482,6 +482,11 @@ def test_groups(desc: GroupTestDesc, mocker, tmp_path):
     compare_nx_graph_with_reference(nx_graph, path_to_dot, sort_dot_graph=False)
 
     filtered_groups = select_largest_groups(not_filtered_groups)
+    print(filtered_groups)
+
+    for x in filtered_groups:
+        print(x)
+    # print(desc.ref_groups)
```
Comment (ljaljushkin) [inline tests/torch/pruning/experimental/test_nodes_grouping.py:489]: please remove debug prints

**Review by ljaljushkin — APPROVED** [review]
LGTM except `prints`

File: tests/post_training/requirements.txt (hunk 1 above)
Comment (AlexanderDokuchaev) [reply tests/post_training/requirements.txt:5]: This сan lead to a hidden version update that breaks the tests, like in this pr that requited update reference.
In tests better use strict version.

**Review by MaximProshin — APPROVED** [review]
